irida_staramr_results/test_unit/test_validate.py

```diff
@@ -1,20 +1,28 @@
 import unittest
 
-from irida_staramr_results import cli
+from irida_staramr_results import validate
 
 
-class TestCli(unittest.TestCase):
+class TestValidate(unittest.TestCase):
+
+    def setup(self):
```

Comment:
Same with this one, change to `setUp(self)` and remove the `self.setup()` line from the tests.

Reply:
Done